projects/packages/videopress/src/class-jwt-token-bridge.php

```diff
@@ -32,8 +32,7 @@ public static function init() {
 			return;
 		}
 
-		add_action( 'admin_enqueue_scripts', array( __CLASS__, 'enqueue_jwt_token_bridge' ), 1 );
-
+		add_action( 'init', array( __CLASS__, 'enqueue_jwt_token_bridge' ), 1 );
```

Comment:
```suggestion
		// Expose the VideoPress token to the Block Editor context, including the front-end editor.
		add_action( 'enqueue_block_assets', array( __CLASS__, 'enqueue_jwt_token_bridge' ), 1 );
		add_action( 'enqueue_block_editor_assets', array( __CLASS__, 'enqueue_jwt_token_bridge' ), 1 );

		// Expose the VideoPress token to the WPAdmin context.
		add_action( 'admin_enqueue_scripts', array( __CLASS__, 'enqueue_jwt_token_bridge' ), 1 );
```

This change would make it work on the front-end editor and third-party editors too.